specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/HybridCompute.json

```diff
@@ -962,68 +854,85 @@
           },
           {
-            "name": "location",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "description": "The location of the Extension being received."
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
           },
           {
-            "name": "publisher",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "description": "The publisher of the Extension being received."
+            "$ref": "#/parameters/machineNameParameter"
           },
           {
-            "name": "extensionType",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "description": "The extensionType of the Extension being received."
+            "$ref": "#/parameters/licenseProfileNameParameter"
           }
         ],
         "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/ExtensionValueListResult"
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Location": {
+                "description": "The URL of the resource used to check the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
+                "type": "integer",
+                "format": "int32"
+              },
+              "Azure-AsyncOperation": {
+                "description": "The URI to poll for completion status.",
+                "type": "string"
+              }
             }
           },
+          "204": {
+            "description": "No Content"
+          },
           "default": {
-            "description": "Error in retrieving extension list",
+            "description": "Error response describing why the operation failed.",
             "schema": {
               "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
             }
           }
         },
-        "x-ms-pageable": {
-          "nextLinkName": null
-        },
+        "x-ms-long-running-operation": true,
         "x-ms-examples": {
-          "GET a list of extensions": {
-            "$ref": "./examples/ExtensionMetadata_List.json"
+          "Delete a License Profile": {
+            "$ref": "./examples/licenseProfile/LicenseProfile_Delete.json"
           }
         }
       }
     },
-    "/providers/Microsoft.HybridCompute/operations": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HybridCompute/machines/{machineName}/licenseProfiles": {
```

Comment:
we don't actually have this api.

Reply:
The pipeline requires every get single resource has a collection list